mmv1/products/memorystore/Instance.yaml

```diff
@@ -193,6 +193,38 @@ properties:
             description:
               "Output only. Target node type for the instance."
             output: true
+  - name: 'automatedBackupConfig'
+    type: NestedObject
+    send_empty_value: true
```

Comment:
is the reason for the `send_empty_value: true` here so that users can explicitly set an empty config here?